style/battle.css

```diff
@@ -315,11 +315,25 @@ License: GPLv2
 	position: absolute;
 	left: 0;
 	right: 0;
-	bottom: 102px;
+	bottom: 75px;
 }
 .rightbar .trainer {
 	bottom: 232px;
 }
+
+.trainer-p1 {
+	bottom: 75px;
+}
+.trainer-p2 {
+	bottom: 232px;
+}
+.trainer-p3 {
+	bottom: 232px;
+}
+.trainer-p4 {
+	bottom: 75px;
```

Comment:
This doesn't work. Seems like the `.rightbar .trainer` above is overriding this one.

Reply:
You can just remove the `.rightbar .trainer`; it's redundant with your changes here, right?

Reply:
Did it work? It looks like it worked.